requirements.txt

```diff
@@ -1,5 +1,5 @@
 cffi==1.9.1
-cryptography==1.7.1
+cryptography==1.7.2
```

Comment:
In fact, in this piece of work unless there's a security advisory we should avoid bumping versions and where we're behind raise a ticket to address it across the services. While we're trying to test the new deployment things need to stay like for like as much as possible.